src/ReportManager/SchemaFormContext/utils.js

```diff
@@ -0,0 +1,8 @@
+
+export const isSchemaFieldRequired = (schema, fieldName) => schema.json.required.includes(fieldName);
+
```

Comment:
The utils methods should not have any external dependencies otherwise than external modules or constants. That way there wouldn't be any circular dependencies. And everyone should import these methods from here instead of expect them to come from context 👍